ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/management-api-gnmi.yml

```diff
@@ -1,9 +1,20 @@
+---
 management_api_gnmi:
   provider: "eos-native"
   transport:
+    grpc:
+    - name: MGMT
+      ssl_profile: gnmi
+      vrf: MGMT
+      notification_timestamp: "send-time"
+      ip_access_group: acl1
```

Comment:
are we not losing in coverage there as we don't have any more list entry without  `notification_timestamp`?